Mergin/utils.py

```diff
@@ -399,14 +399,13 @@ def send_logs(username, logfile):
 
 def validate_mergin_url(url):
     """
-    Validation of mergin URL by pinging. Checks if URL points at compatible Mergin server.
+    Initiates connection to the provided server URL to check if the server is accessible
     :param url: String Mergin Maps URL to ping.
     :return: String error message as result of validation. If None, URL is valid.
     """
     try:
-        mc = MerginClient(url, proxy_config=get_qgis_proxy_config(url))
-        if not mc.is_server_compatible():
-            return "Incompatible Mergin Maps server"
+        _ = MerginClient(url, proxy_config=get_qgis_proxy_config(url))
```

Comment:
The python convention is to use `_` as a throwaway when we absolutely need place holder variable (e.g unpack variable), here I think we can just remove it altogether

```suggestion
        MerginClient(url, proxy_config=get_qgis_proxy_config(url))
```